content/en/docs/components/multi-tenancy/overview.md

```diff
@@ -38,4 +38,4 @@ When configuring multi-user isolation along with your security and identity mana
 
 ## Next steps
 
-* Learn [how more on multi-user isolation and profiles](/docs/components/multi-tenancy/getting-started/).
+* Learn [how to design for multi-user isolation](/docs/components/multi-tenancy/design/).
```

Comment:
nit: do you think "Learn more about [multi-user isolation design]" describes the link better?

Reply:
sgtm, updated!